tests/model_serving/model_server/kserve/negative/test_empty_body.py

```
@@ -0,0 +1,104 @@
+    assert_pods_healthy,
+    send_inference_request,
+)
+
+pytestmark = pytest.mark.usefixtures("valid_aws_config")
+
+# Both 400 (Bad Request) and 412 (Precondition Failed) are acceptable for empty
+# bodies; OVMS often returns 412 when it cannot parse the body as KServe v2 JSON.
+EMPTY_BODY_EXPECTED_CODES: set[int] = {
+    HTTPStatus.BAD_REQUEST,          # 400
+    HTTPStatus.PRECONDITION_FAILED,  # 412
+    HTTPStatus.LENGTH_REQUIRED,      # 411 – some proxies require Content-Length > 0
```

Ruff (0.15.20)

[warning] Comment contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF003)